support/mender-inventory-hostinfo

```diff
@@ -11,9 +11,6 @@ export LC_ALL
 grep 'model name' /proc/cpuinfo | uniq | awk -F': ' '
      // { printf("cpu_model=%s\n", $2);}
 '
-echo "cpu_count=$(ls /sys/bus/cpu/devices/ | wc -l)"
```

Comment:
This was one of the things mentioned in acceptance criteria: https://tracker.mender.io/browse/MEN-928